README.md

```diff
@@ -29,12 +29,12 @@ software.
 
 The Astrobee Robot Software is in a beta stage. This means that some
 features are incomplete, and extensive changes can be expected. Please consult the
-[release](RELEASE.md) for the current list of features and limitations.
+[release](https://nasa.github.io/astrobee/html/md_RELEASE.html) for the current list of features and limitations.
 
 ### Usage instructions
 
 To install and use astrobee, please see the
-[usage instructions](INSTALL.md).
+[usage instructions](https://nasa.github.io/astrobee/html/md_INSTALL.html).
```

Comment:
Looks good! Should we remove the astrobee/Install.md file so people don't go there by accident?

Reply:
The documentation is generated using doxygen which uses all the .md files, if you delete that file you also delete it in the documentation.